src/adapter.js

```diff
@@ -280,6 +282,12 @@ class MongooseAdapter {
 
     await CasbinRule.deleteMany(where);
   }
+
+  async close () {
+    if (this.mongoseInstance && this.mongoseInstance.connection) {
```

Comment:
Afterwards, if you will have a code like `once('connected', () => this.connection = connection)`, here you could write simpler `if (this.connection) { close }`

Reply:
Thanks!